atomic_reactor/util.py

```diff
@@ -829,6 +829,18 @@ def get_manifest_list(self, image):
         response, _ = self.get_manifest(image, version)
         return response
 
+    def get_manifest_list_digest(self, image):
```

Comment:
I really thought we already had a method for this somewhere. Could not find one though...